TLM/TLM/U/HelperExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace TrafficManager.U {
+    using ColossalFramework.UI;
+    using UnityEngine;
+
+    public static class HelperExtensions {
+        /// <summary>
+        /// Get rectangle of a control in GUI screen space.
+        /// GUI screen space is used to draw overlay textures in overlay helpers.
+        /// </summary>
+        public static Rect GetScreenRectInGuiSpace(this UIComponent ctrl) {
+            Vector3 absPosition = ctrl.absolutePosition;
+            float scaleX = Screen.width / 1920f;
+            float scaleY = Screen.height / 1080f;
+            return new Rect(
+                absPosition.x * scaleX,
+                absPosition.y * scaleX,
+                ctrl.width * scaleY,
+                ctrl.height * scaleY);
+            // var b = ctrl.GetBounds();
```

Comment:
did you forget to delete commented out code?

Reply:
Deleted now. This class is currently shown in grey as unused.
It will be used after future batches are merged.